packages/eslint-plugin-pf-codemods/lib/rules/v5/table-warn-actionsColumn.js

```diff
@@ -0,0 +1,21 @@
+// https://github.com/patternfly/patternfly-react/pull/8629
+module.exports = {
+  create: function (context) {
+    return {
+      ImportDeclaration(node) {
+        const TableImport = node.specifiers.find(
+          (specifier) =>
+            (specifier.imported.name === "Table" || specifier.imported.name === "TableComposable") &&
+            node.source.value === "@patternfly/react-table"
+        );
+
+        if (TableImport) {
+          context.report({
+            node,
+            message: "The ActionsColumn now uses the new version of Dropdown. The action toggle should now pass a MenuToggle rather than a DropdownToggle, and direction and position properties are now passed under the new popperProps property.",
```

Comment:
Nit to go along with the rebase if you think it'd be worth updating:

```suggestion
            message: "The ActionsColumn now uses the Next version of Dropdown. The action toggle should now pass a MenuToggle rather than a DropdownToggle, and direction and position properties are now passed under the new popperProps property.",
```